src/Sprite/TilesetLoader.h

```diff
@@ -25,6 +26,10 @@ namespace Tileset
 	// After Reading into memory, reformats into standard 8 bit indexed bitmap before returning results
 	BitmapFile ReadCustomTileset(Stream::Reader& reader);
 
+	// Write tileset in Outpost 2's custom bitmap format.
+	// To write tileset in standard bitmap format, use BitmapFile::WriteIndexed
+	void WriteCustomTileset(Stream::Writer& writer, const BitmapFile& tileset);
```

Comment:
Should we make this a member method? I suppose the case is less clear here, since we might consider this a format conversion to effectively a new type, or we might consider it an alternate serialization format of the same type. Since we only have a single bitmap class, I might be inclined to use a member method and treat this as an alternate serialization format for a single type.

Reply:
Hmm, I hadn't thought about that. I figured it belonged with the rest of the Tileset code. Placing it inside of BitmapFile would make the calls to BitmapFile member variables a bit simpler and support better encapsulation in the future.